library/cartridge_get_instance_info.py

```diff
@@ -31,6 +31,12 @@ def get_instance_systemd_service(app_name, instance_name, stateboard=False):
     return '%s@%s' % (app_name, instance_name)
 
 
+def get_instance_systemd_service_template(app_name, stateboard=False):
```

Comment:
I don't understand why this function name contains "template"? It returns systemd service file name, isn't it?

Reply:
It returns path to systemd template file (without instance name, with %i variable). `get_instance_systemd_service` returns path to instance systemd service